src/configurationProvider.ts

```diff
@@ -88,29 +88,27 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
                 await updateDebugSettings();
             }
 
-            if (Object.keys(config).length === 0) { // No launch.json in current workspace.
-                // check whether it is opened as a folder
-                if (folder !== undefined) {
-                    // for opened with folder, return directly.
-                    return config;
-                }
-                // only rebuild for single file case before the build error issue is resolved.
+            // just create and open the lanuch.json if there is no lanch.json in current folder

```

Comment:
Change the comment to `If no launch.json exists in the current workspace folder, delegate to provideDebugConfigurations api to generate the initial launch.json configurations.`